.travis.yml

```diff
@@ -7,6 +7,7 @@ cache:
     - ./Unity
 before_install:
 - chmod a+x ./Travis/installUnity.sh
+- npm install npm@latest -g
```

Comment:
Was using `npm@latest` recommended in `semantic-release` documentation? To me, that seems inadvisable since that could introduce breaking changes if `npm` releases a new major version.

Reply:
Yeah I followed the guide [here](https://github.ibm.com/MIL/watson-swagger-codegen/wiki/Incorporating-semantic-release-into-your-SDK#adding-integration). Should I just leave this command out? or what should i replace it with

Reply:
Yeah I might've been wrong to add that. I remember Dustin bringing it up before.

It looks like you can replace it with `npm install npm@6` to get the newest version and stay on major version 6.

Reply:
It's really not a big deal but I would use `npm install -g npm@6.x` to future-proof yourself